### PR TITLE
Fixes #25 - PHPSESSID is giving a SameSite error on LTI launch

### LINKED ISSUE #24: Remove configuration that changes lax to none as it's no longer needed
In the most recent version of the code this isn't needed so these lines can be removed.

https://github.com/tl-its-umich-edu/docker-base-images/blob/d5c325cb2a0a35196b6c2016bb3b0fb8658e2f68/qualtrics-lti/Dockerfile.umich#L5

### LINKED ISSUE #25: PHPSESSID is giving a SameSite error on LTI launch
When the tool is launched in an iframe with LTI it's giving an error about PHPSESSID being unsecure. 

I found this writeup about a few ways to resolve this. https://stackoverflow.com/questions/60157086/php-samesite-session-problem-session-doesnt-work

It doesn't look like this session is being set by this app, so it must be in some package?

### PR DESCRIPTION
**Pull request by jonespm**
Also fixes #24 - Remove configuration that changes lax to none as it's no longer needed

### CODE REVIEW

File: qualtrics-lti/Dockerfile.umich [hunk 1]
```diff
@@ -1,8 +1,10 @@
 FROM ghcr.io/longhornopen/qualtrics-lti:20230215
 
-WORKDIR /var/www/html
-# Update lax to none in the session config
-RUN sed -i s/lax/none/ config/session.php
+WORKDIR /var/www/html/
+
+# Add our .htaccess into the one there
+COPY .htaccess.append .
+RUN cat .htaccess.append >> public/.htaccess && rm .htaccess.append
```
Comment (lsloan) [inline qualtrics-lti/Dockerfile.umich:7]: Nice technique!

**Review by lsloan — APPROVED** [review]
Seems pretty straightforward.  :shipit:

File: qualtrics-lti/Dockerfile.umich (hunk 1 above)
Comment (jonespm) [reply qualtrics-lti/Dockerfile.umich:7]: Yeah, it's not ideal, [I found it here](https://github.com/moby/moby/issues/12193) and seems to be the best way to append right now.

**Comment by jonespm** [comment]
It doesn't look like these changes are going to be needed with the latest release, so closing.

**Pull request closed without merging**